core/station/api/spec.did

```diff
@@ -2315,6 +2327,18 @@ type ExternalCanisterPermissions = record {
 // The create input type for setting the permissions for the external canister.
 type ExternalCanisterPermissionsCreateInput = ExternalCanisterPermissions;
 
+// The pair that is used to represent the execution and validation method.
+type CanisterExecutionAndValidationMethodPair = record {
+  // The validation method that is used to validate the request and
+  // render the argument.
+  validation_method : ValidationMethodResourceTarget;
+  // The method that the caller can call on the external canister.
+  //
+  // The `*` method name is used to represent that the caller can
+  // call any method on the canister.
+  execution_method : text;
```

Comment:
Could we use a typed struct instead of text to represent the case of Any (instead of using `*` for that)?

Reply:
this is irrelevant if we go for the approach from this [comment](https://github.com/dfinity/orbit/pull/381#discussion_r1793839137)

Reply:
i would rather not, this is consistent to many other fields we already have for `execution_method` in `ExternalCanisterCallRequestPolicyRuleInput`, `ExternalCanisterCallRequestPolicyRule`, `ExternalCanisterCallPermission` and `ExternalCanisterCallerMethodsPrivileges`.

Reply:
However, `*` is also a valid IC canister method name (I just checked by deploying a canister with such a method and calling it via `dfx`).

Reply:
Even if supported by a canister imo we should keep as is to avoid a possible migration, refactor at the moment and if someone has a need for it in the future we can revisit the decision.